### DISCUSSION TITLE
Replacing svg PrimeReact Sort Icons with Custom Icons

### DISCUSSION THREAD

**Discussion by tewabet** [primefaces/community#289] [PrimeReact] [answered]
I am hopeful that someone here can offer guidance or insights into my current challenge.

**Issue Description:**
In PrimeReact, the sorting icons are currently implemented using SVG. I am looking to completely replace these SVG icons with my own custom icons.

**Steps Taken:**
So far, I have employed custom CSS styles to visually alter the icons, utilizing properties like 'display: none.' While this approach allows for a visual alternative, I am seeking a way to entirely remove the SVG icons.

**Request for Assistance:**
My primary objective is to completely remove the SVG icons and implement my own custom icons in their place. I would greatly appreciate any assistance or guidance on achieving this. If anyone can provide insights or a step-by-step approach to remove the SVG icons and integrate custom ones, it would be immensely helpful.

Thank you in advance for your time and consideration. I eagerly anticipate hearing from the esteemed members of the PrimeReact community and benefiting from your wealth of experience.

**Comment by habubey** [comment]
Hi there, SVG takes its icons from primeicons, my suggestion to you is to remove primeicons from the dependencies, but give them separately to each component you use.
He leaves a link below where you can take his classes.

https://codesandbox.io/s/primereact-demo-forked-4mt45s?file=/src/App.js

**Comment by melloware** [comment, marked as answer]
Here is a working example replacing the Sort icons of the Datatable as well: https://stackblitz.com/edit/react-h8t1sm?file=src%2FApp.js
